src/topp/FalseDiscoveryRate.cpp

```diff
@@ -89,8 +89,21 @@ class TOPPFalseDiscoveryRate :
     setValidStrings_("PSM", ListUtils::create<String>("true,false"));
     registerStringOption_("peptide", "<FDR level>", "false", "Perform FDR calculation on peptide level and annotates it as meta value\n(Note: if set, also calculates FDR/q-value on PSM level.)", false);
     setValidStrings_("peptide", ListUtils::create<String>("true,false"));
+    registerStringOption_("PSM_peptide_base_score", "<score name or type>", "", "Set if you want to choose a different score than the last calculated main score for PSM or peptide level.", false);
+    registerStringOption_("PSM_peptide_base_score_orientation", "<higher/lower>", "", "In case the score orientation cannot be inferred.", false, true);
+    setValidStrings_("PSM_peptide_base_score_orientation", ListUtils::create<String>("higher_better, lower_better"));
     registerStringOption_("protein", "<FDR level>", "true", "Perform FDR calculation on protein level", false);
     setValidStrings_("protein", ListUtils::create<String>("true,false"));
+    registerStringOption_("protein_group", "<FDR level>", "true", "Perform FDR calculation on (indist.) protein group level, too. Currently, this will enable protein FDR automatically (since internals need to be in-sync) but will affect the level at which it filters (if enabled).", false);
```

Comment:
This probably should be false by default , or should only have an effect if protein is also true.